lib/registry.ts

```diff
@@ -65,6 +65,7 @@ export interface ICaseHook {
   position?: Position;
   tags?: string;
   name?: string;
+  order?: number;
```

Comment:
Make this required and give it a default value during assignment.

Reply:
Makes sense, addressed this